iamkanhaiyakumar/ai-content-generator
# Implemented Docker integration (Closes #15)

## What does this PR do?

This PR **adds a Docker setup** for the application to facilitate containerization and deployment. It addresses the issue of inconsistent development environments and streamlines the build process.

Fixes #15 

## Type of change


## How should this be tested?

To verify the changes, please follow these steps:

1. Clone the repository and navigate to the project directory.
2. Run `docker-compose up` to build and start the application in a Docker container.
3. Ensure that the application runs without errors and is accessible at `http://localhost:3000`.

### Tests to run:

- [ ] Check if the application loads successfully.
- [ ] Validate that all existing features function as expected within the Docker environment.

## Mandatory Tasks
